content/headless/opentelemetry-authentication.md

```diff
@@ -0,0 +1,52 @@
+---
+---
+## `auth()` {#auth}
+
+You can set authentication in the `auth()` option of the `opentelemetry()` driver. By default, authentication is disabled (`auth(insecure())`).
+
+The following authentication methods are available in the `auth()` block:
+
+### `adc()` {#adc}
+
+[Application Default Credentials (ADC)](https://cloud.google.com/docs/authentication/application-default-credentials). This authentication method is only available for destination. It accepts the `target-service-account()` option, where you can list service accounts to match against when authenticating the server.
+
+### `alts()` {#alts}
+
+[Application Layer Transport Security (ALTS)](https://grpc.io/docs/languages/cpp/alts/) is a simple to use authentication, only available within Google's infrastructure.
+
+```shell
+source {
+    opentelemetry(
+      port(12345)
+      auth(alts())
+    );
+  };
+```
+
+### `insecure()` {#insecure}
+
+This is the default method, authentication is disabled (`auth(insecure())`).
+
+### `tls()` {#tls}
+
+<!-- FIXME xinclude these from the other tls blocks -->
```

Comment:
[note]

The `opentelemetry()` drivers have a completely different `tls()` block implementation from the `network()` or `http()` drivers. Currently it has most of the features, but it is not guaranteed to have 100% feature parity.

Reply:
@eldarnash Oh I meant to write this note to you only, not to put it to the docs, too, however it is okay to have it there. I will make sure to differentiate between a doc note and a note to the author when writing a comment. Sorry!